src/OSPSuite.Core/Extensions/EnumerableExtensions.cs

```diff
@@ -60,5 +60,50 @@ public static IReadOnlyList<T> Complement<T>(this IEnumerable<T> enumeration1, I
          var list2 = enumeration2.ToList();
          return list1.Union(list2).Except(list1.Intersect(list2)).ToList();
       }
+
+      /// <summary>
+      ///    Returns the index of the first occurrence in a sequence
+      /// </summary>
+      /// <typeparam name="TSource">The type of the elements of source.</typeparam>
+      /// <param name="list">A sequence in which to locate a value.</param>
+      /// <param name="value">The object to locate in the sequence</param>
+      /// <returns>The zero-based index of the first occurrence of value within the entire sequence, if found; otherwise, �1.</returns>
+      public static int IndexOf<TSource>(this IEnumerable<TSource> list, TSource value)
```

Comment:
why are we redefined the method fro m the framework? Isn't index of using equal?